File: CS/WindowsFormsApplication1/ReportPreviewHelper.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraReports.UI;

namespace WindowsFormsApplication1 {
    static class ReportPreviewHelper {
        public static void ShowReport(Form mdiContainer) {
            XtraReport1 report = new XtraReport1();
            ReportPrintTool printTool = new ReportPrintTool(report);
            printTool.PreviewRibbonForm.MdiParent = mdiContainer;
            printTool.PreviewRibbonForm.WindowState = FormWindowState.Maximized;
            printTool.ShowRibbonPreview();
        }
    }
}

File: VB/WindowsFormsApplication1/ReportPreviewHelper.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraReports.UI

Namespace WindowsFormsApplication1
    Friend NotInheritable Class ReportPreviewHelper

        Private Sub New()
        End Sub

        Public Shared Sub ShowReport(ByVal mdiContainer As Form)
            Dim report As New XtraReport1()
            Dim printTool As New ReportPrintTool(report)
            printTool.PreviewRibbonForm.MdiParent = mdiContainer
            printTool.PreviewRibbonForm.WindowState = FormWindowState.Maximized
            printTool.ShowRibbonPreview()
        End Sub
    End Class
End Namespace
